### ISSUE TITLE
Custom rules don't work ('Unsupported rule' errors)

### ISSUE THREAD

**Issue by anikolaev** [dockyard/ember-suave#38]
I have several custom rules in lib/jscs-rules folder and the following line in .jscsrc file:
```
  "additionalRules": [ "lib/jscs-rules/*.js" ],
```
All custom rules are activated in config.

When I run server in ember-cli it gives me many errors like these:
```
Unsupported rule: disallowMultipleVarDeclWithAssignment at unit/pods/components/report-table/cell/component-test.js :
     1 |import { moduleForComponent, test } from 'ember-qunit';
--------^
     2 |
     3 |moduleForComponent('report-table/cell', 'Unit | Component | report table/cell', {
Unsupported rule: disallowMultipleVarDeclWithAssignment at unit/pods/components/report-table/component-test.js :
     1 |import { moduleForComponent, test } from 'ember-qunit';
--------^
     2 |
     3 |moduleForComponent('report-table', 'Unit | Component | report table', {
Unsupported rule: disallowMultipleVarDeclWithAssignment at unit/pods/index/controller-test.js :
     1 |import { moduleFor, test } from 'ember-qunit';
--------^
     2 |
     3 |moduleFor('controller:index', {
```

The path is correct because if I make a copy of some rule it says about duplicated rules. I also noticed that rules from node_modules/ember-suave/lib/rules work correctly but when I simply move some rule like disallow-space-before-semicolon.js to my rules folder I get errors:

```
Unsupported rule: disallowSpacesBeforeSemicolons at unit/pods/report/controller-test.js :
     1 |import { moduleFor, test } from 'ember-qunit';
--------^
     2 |
     3 |moduleFor('controller:report', {
Unsupported rule: disallowSpacesBeforeSemicolons at unit/pods/report/route-test.js :
     1 |import { moduleFor, test } from 'ember-qunit';
--------^
     2 |
     3 |moduleFor('route:report', 'Unit | Route | report', {
Unsupported rule: disallowSpacesBeforeSemicolons at unit/transforms/object-test.js :
     1 |import {
--------^
     2 |  moduleFor,
     3 |  test
```

When I have the same rule in both folders I don't have duplication errors and rule works correctly.

My config: Windows 7, ember-suave: "0.1.9", node.js: v0.12.4, npm: 2.10.1, ember-cli: 0.2.5

**Comment by brzpegasus** [comment]
Thanks for reporting! I will check it out tonight or this weekend.

**Comment by brzpegasus** [comment]
@anikolaev This is fixed and published in v0.1.10.

**Comment by anikolaev** [comment]
Great, thank you.
